extending/template_system/info_graph.rst

```diff
@@ -0,0 +1,178 @@
+.. _template_info_graph:
+
+
+Graph output for :command:`conan info` command
+==============================================
+
+.. warning::
+
+    This has to be an considered as an **experimental** feature, we might
+    change the context provided to this templates once we have more exmpales
+    from the community.
+
+
+The :ref:`conan_info` command can generate a visualization of the dependency graph, it
+comes in two flavors: `html` and `dot` (GraphViz), but both take the same template
+parameters. Conan will use the following input files, if found, inside the Conan
+cache folder:
+
+ * HTML output: *<cache>/templates/output/info_graph.html*.
+ * DOT output: *<cache>/templates/output/info_graph.dot*.
+
+Context
+-------
+
+These files should be valid Jinja2 templates and they will be feed with the following
```

Comment:
may we have a link to Jinja2 syntax documentation here, so user can get familiar with Jinja2, its syntax and how to write templates

Reply:
Added a link to every section on the first occurrence of the Jinja2 word